Header header

# like the output of posix 'free', in bytes
uint64    used  
uint64    total
uint64    free
uint64    shared
uint64    buff_cache
uint64    available